lib/packwerk/offense.rb

```diff
@@ -22,9 +29,11 @@ def initialize(file:, message:, location: nil)
 
     sig { params(style: OutputStyle).returns(String) }
     def to_s(style = OutputStyles::Plain.new)
-      if location
+      # So sorbet knows it's non-nil after the if check
```

Comment:
No need for this comment. If people try to remove they will know why it is there.

Reply:
Makes sense